src/models/db/conditionFulfillment.js

```diff
@@ -23,14 +20,21 @@ class ConditionFulfillment extends Model {
   static convertFromPersistent (data) {
     delete data.created_at
     delete data.updated_at
+    if (typeof data.condition_fulfillment === 'string') {
```

Comment:
Perhaps adding a line of comment to explain why we'd like to selectively convert a string to JSON might help?